Create spring-plugin.travis.yml

asellappen opened the pull request.

gururajrkatti reviewed (commented):
It is a small plugin/ test. Do we require travis_wait here ? Rest all LGTM

asellappen commented:
> It is a small plugin/ test. Do we require travis_wait here ? Rest all LGTM

Thats true ,I have added it for safe due to vm slowness.
Removed now,pl check.

gururajrkatti reviewed (approved):
ok
